waku/waku_archive/common.nim

```diff
@@ -56,6 +56,7 @@ type
   ArchiveResponse* = object
     hashes*: seq[WakuMessageHash]
     messages*: seq[WakuMessage]
+    topics*: seq[PubsubTopic]
```

Comment:
Mmm. Wouldn't it be better to return seq[(message, topic)] to keep the 1:1 association explicit?